HW5/premium.py

```diff
@@ -0,0 +1,7 @@
+a = ['ivan', 'petr', 'abdul']
+b = [10000, 20000, 30000]
+c = ['9.25%', '11%', '0.5%']
+
+d = {i: j * float(k.replace('%',''))/100 for i, j, k in zip(a,b,c) }
```

Comment:
Можно не использовать метод `replace` есть инструмент куда проще и более знакомый.
```suggestion
d = {i: j * float(k[:-1]) / 100 for i, j, k in zip(a, b, c) }
```
Кроме того, следование PEP8 никто не отменял.